cloud/deploy/deploy_test.go

```diff
@@ -585,6 +585,10 @@ func TestNoDagsDeployVR(t *testing.T) {
 }
 
 func TestDeployFailure(t *testing.T) {
+	os.Mkdir("./testfiles/dags", os.ModePerm)
+	path := "./testfiles/dags/test.py"
+	fileutil.WriteStringToFile(path, "testing")
+
```

Comment:
```suggestion
                defer os.RemoveAll("./testfiles/dags/")
```
better to have defer right after creation, so that even if something fails in between the dags file are always deleted

Reply:
Updated